frontend/jsontablemodel.cpp

```diff
@@ -69,81 +69,45 @@ QJsonObject JsonTableModel::getJsonObject(const QModelIndex &index) const
 // Fills table view
 QVariant JsonTableModel::data(const QModelIndex &index, int role) const
 {
-    switch (role)
+    if (role != Qt::DisplayRole)
+        return QVariant();
```

Comment:
Small C++ tip - when returning a default-constructed value like this you can say just `return {};` to avoid repeating the type.